apps/frontend/src/pages/SearchPage.tsx

```diff
@@ -44,14 +44,22 @@ const SearchPage = () => {
   }
 
   return (
-    <div className="justify-center">
-      <ProfileButton></ProfileButton>
-      <HistoryButton callback={isOpened => setSidebarOpened(isOpened)}></HistoryButton>
-      <Sidebar isOpened={isSidebarOpened} history={history} activePage={activePage} setActivePage={setActivePage}></Sidebar>
+    <div className={"flex flex-col duration-300 h-screen bg-(--color-white-background) " + (isSidebarOpened && "ml-100")}>
+      <div className="flex justify-between p-5">
+        <div>
+          <HistoryButton toggleCallback={isOpened => setSidebarOpened(isOpened)} stateOpened={false}></HistoryButton>
```

Comment:
This can be simpler:
```suggestion
          <HistoryButton toggleCallback={setSidebarOpened} stateOpened={false}></HistoryButton>
```